include/ensmallen_bits/sgdr/sgdr.hpp

```diff
@@ -71,11 +72,12 @@ class SGDR
    *        parameters.
    * @param resetPolicy If true, parameters are reset before every Optimize
    *        call; otherwise, their values are retained.
+   * @param stepSizeMin Final step size before each restart
    */
-  SGDR(const size_t epochRestart = 50,
+  SGDR(const double stepSizeMax = 0.01,
+       const size_t epochRestart = 50,
```

Comment:
Why has `stepSizeMax` suddenly become the first parameter?  And where is `stepSizeMin`?